modules/serverless-install-serving-web-console.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+//  * serverless/installing_serverless/installing-knative-serving.adoc
+
+[id="serverless-install-serving-web-console_{context}"]
+= Installing Knative Serving using the web console
+
+.Procedure
+. Navigate in the web console to the *Catalog* → *OperatorHub* → *Installed Operators* page.
+. Check that the *Project* dropdown at the top of the page is set to *Project: knative-serving*.
+. Click *Knative Serving* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Serving* tab.
++
+image::serverless-installed-operator.png[Installed Operators page]
+. Click the *Create Knative Serving* button.
++
+image::serverless-create-serving.png[Knative Serving tab]
+. In the *Create Knative Serving* page, you can choose to configure the `KnativeServing` object by using either the default form provided, or by editing the YAML.
++
+* Using the form is recommended for simpler configurations that do not require full control of `KnativeServing` object creation.
++
+If you are configuring the `KnativeServing` object using the form, make any changes that you want to implement for your Knative Serving deployment (optional), then click *Create*.
```

Comment:
Minor - the word "(optional)" threw me off here. I suggest moving it to the beginning of the step, and breaking "Click Create." into a new step.